src/openMVG/cameras/Camera_Intrinsics.hpp

```diff
@@ -128,7 +128,7 @@ struct IntrinsicBase
 };
 
 /// Return the angle (degree) between two bearing vector rays
-static double AngleBetweenRay(
+static inline double AngleBetweenRay(
```

Comment:
What about if we move the implementations (for the functions that are not templatized) more properly into a cpp file? We should not need static anymore and fix the multiple definition, no?